File: app/src/main/java/com/xxxifan/smsmaid/SQLCipher.java
package com.xxxifan.smsmaid;

import com.raizlabs.android.dbflow.config.DatabaseDefinition;
import com.raizlabs.android.dbflow.sqlcipher.SQLCipherOpenHelper;
import com.raizlabs.android.dbflow.structure.database.DatabaseHelperListener;

/**
 * Created by BobPeng on 2017/2/18.
 */

public class SQLCipher extends SQLCipherOpenHelper {
    public SQLCipher(DatabaseDefinition databaseDefinition, DatabaseHelperListener listener) {
        super(databaseDefinition, listener);
    }

    @Override
    protected String getCipherSecret() {
        return "xxxifan";
    }
}

File: app/src/main/java/com/xxxifan/smsmaid/SQLCipher.kt
package com.xxxifan.smsmaid

import com.raizlabs.android.dbflow.config.DatabaseDefinition
import com.raizlabs.android.dbflow.structure.database.DatabaseHelperListener
import com.raizlabs.dbflow.android.sqlcipher.SQLCipherOpenHelper

/**
 * Created by BobPeng on 2017/2/10.
 */
class SQLCipher(database: DatabaseDefinition, listener: DatabaseHelperListener) : SQLCipherOpenHelper(database, listener) {

    override fun getCipherSecret(): String = "xxxifan"

}